File: frontend/ReactMusicCommerce/src/models/AdminOrderDTO.ts
export interface AdminOrderDTO {
  orderId: number;
  userId: number;
  customerName: string;
  customerEmail: string;
  totalAmount: number;
  paymentStatus: string;
  paymentMethod: string;
  gatewayTransactionCode: string | null;
  createdAt: string;
  totalItems: number;
}

File: backend/music/src/main/java/com/springboot/music/dto/AdminOrderDTO.java
package com.springboot.music.dto;

import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AdminOrderDTO {

    private Integer orderId;
    private Integer userId;
    private String customerName;
    private String customerEmail;
    private Double totalAmount;
    private String paymentStatus;
    private String paymentMethod;
    private String gatewayTransactionCode;
    private LocalDateTime createdAt;
    private Integer totalItems;
}

